src/unit/Batter.js

```diff
@@ -0,0 +1,20 @@
+import inputValidation from "../validify/inputValidation.js";
+import {Console} from "@woowacourse/mission-utils";
+
+class Batter {
+    constructor() {
+        this.gameNumbers = [];
+    }
+
+    async enterGameNumbers () {
+        try {
+            const batterInput = await Console.readLineAsync("숫자를 입력해주세요 : ");
+            this.gameNumbers = batterInput.split("").map(Number);
```

Comment:
사소하지만 split 대신 [...] 스프레드 연산자를 사용해보면 어떨까 하는 의견입니당!  [참고](https://velog.io/@sunohvoiin/JavaScript-%EB%AC%B8%EC%9E%90%EC%97%B4%EC%9D%84-%EB%B0%B0%EC%97%B4%EB%A1%9C-%EB%B3%80%ED%99%98%ED%95%98%EB%8A%94-3%EA%B0%80%EC%A7%80-%EB%B0%A9%EB%B2%95-convert-string-to-array)
개인적으로 가독성도 더 좋고, 속도도 더 빠르고, 이모티콘 👀 까지 처리할 수 있어서 좋다고 느꼈어요! 

```suggestion
            this.gameNumbers = [...batterInput].map(Number);
```

https://velog.io/@sunohvoiin/JavaScript-%EB%AC%B8%EC%9E%90%EC%97%B4%EC%9D%84-%EB%B0%B0%EC%97%B4%EB%A1%9C-%EB%B3%80%ED%99%98%ED%95%98%EB%8A%94-3%EA%B0%80%EC%A7%80-%EB%B0%A9%EB%B2%95-convert-string-to-array

더 나아가서 배열을 매핑할 때는 중간 배열 생성을 방지를 위해 전개 구문 ... 대신 
Array.from을 사용하는 것도 좋은 방법 중 하나인 것 같아요! [참고](https://github.com/ParkSB/javascript-style-guide#arrays--mapping)

Reply:
피드백 감사드립니다!! 확실히 스프레드 연산자가 더 좋아 보이네요!! 가르침 감사드립니다!! :)